### PR TITLE
Add Sentinel-1 Burst InSAR readme file

### PR DESCRIPTION
**Pull request by hjkristenson**
This PR adds a readme file template for S1 Burst InSAR (ISCE2) products.
The basic content follows the readme for InSAR GAMMA products, and TODOs have been added throughout where content needs to be adjusted for use with the ISCE2 burst-based products.

### CODE REVIEW

File: src/hyp3_isce2/metadata/templates/insar_burst/readme.md.txt.j2 [hunk 1]
```diff
@@ -0,0 +1,296 @@
...
+
+To reference HyP3 in manuscripts, cite our documentation available at https://github.com/ASFHyP3/hyp3-docs:
+
+    Hogenson, K., Kristenson, H., Kennedy, J., Johnston, A., Rine, J., Logan, T., Zhu, J., Williams, F., Herrmann,
+    J., Smale, J., & Meyer, F. (2020). Hybrid Pluggable Processing Pipeline (HyP3): A cloud-native infrastructure
+    for generic processing of SAR data [Computer software]. https://doi.org/10.5281/zenodo.4646138
+
+DOIs are also provided for citation when discussing the HyP3 software or plugins:
+* HyP3 processing environment, DOI: [10.5281/zenodo.3962581](https://doi.org/10.5281/zenodo.3962581)
+* HyP3 ISCE2 plugin, DOI: [TODO](TODO)
```
Comment (jtherrmann) [inline src/hyp3_isce2/metadata/templates/insar_burst/readme.md.txt.j2:80]:
```suggestion
* HyP3 ISCE2 plugin, DOI: [10.5281/zenodo.8007397](https://doi.org/10.5281/zenodo.8007397)
```

File: src/hyp3_isce2/metadata/templates/insar_burst/readme.md.txt.j2 [hunk 2]
```diff
@@ -0,0 +1,306 @@
...
+special features. For an overview of the dataset, visit
+https://spacedata.copernicus.eu/collections/copernicus-digital-elevation-model.
+
+***************
+# Product Contents #
+
+The files generated in this process include:
+
+1. Wrapped Interferogram (GeoTIFF)
+2. Unwrapped Interferogram (GeoTIFF, PNG image)
```
Comment (hjkristenson) [inline src/hyp3_isce2/metadata/templates/insar_burst/readme.md.txt.j2:131]: Are we including browse images for the interferograms? I was surprised to see that there's a PNG for the unwrapped but not the wrapped; I would have expected it the other way around.

File: src/hyp3_isce2/metadata/templates/insar_burst/readme.md.txt.j2 [hunk 3]
```diff
@@ -0,0 +1,306 @@
...
+
+
+*See below for detailed descriptions of each of the product files.*
+
+----------------
+## 1. Wrapped Interferogram
+
+The wrapped interferogram is output in GeoTIFF file. Because the fringes are on a wrapped 2-pi scale rather than representing continuous phase difference values, a wrapped interferogram tends to be most useful as a color visualization. This product displays deformation in multiples of half of the sensor wavelength, so areas with fringes that are very close together indicate greater amounts of deformation in the line-of-sight direction from the sensor. Each 2-pi cycle is equivalent to about 2.8 cm in line-of-sight displacement (half the wavelength of the Sentinel-1 sensor, which is about 5.6 cm).
+
+The wrapped interferogram image is generated from the interferogram phase prior to unwrapping. The PNG and KMZ files are tagged with _color_phase, and are 2048 pixels wide. The optional GeoTIFF is tagged with _wrapped_phase.tif
```
Comment (hjkristenson) [inline src/hyp3_isce2/metadata/templates/insar_burst/readme.md.txt.j2:148]: Are we including PNG and KMZ files for the wrapped phase? Is the GeoTIFF version included and/or optional?

File: src/hyp3_isce2/metadata/templates/insar_burst/readme.md.txt.j2 [hunk 4]
```diff
@@ -0,0 +1,306 @@
...
+
+----------------
+
+## 7. DEM used to process the data TODO: Edit as appropriate
+
+This is a copy of the Digital Elevation Model (DEM) layer used for processing. This layer is tagged with _dem.tif, and is a 32-bit float GeoTIFF.
+
+The source DEM has a geoid correction applied before it is used for processing, so height values in this file will differ from the source DEM. The DEM is down-sampled to twice the pixel spacing of the desired InSAR output pixel spacing to smooth it before processing. As such, an 80-m InSAR product requires the DEM to be down-sampled to a pixel spacing of 160 m. It is provided in the same pixel spacing/alignment and projected to the same UTM Zone as the interferogram files.
+
+*This file is optional. Select the "Include DEM" option in the On Demand HyP3 Processing Options to include it in the product package.*
```
Comment (jtherrmann) [inline src/hyp3_isce2/metadata/templates/insar_burst/readme.md.txt.j2:211]: I don't think it's optional for this plugin?

File: src/hyp3_isce2/metadata/templates/insar_burst/readme.md.txt.j2 (hunk 2 above)
Comment (jtherrmann) [reply src/hyp3_isce2/metadata/templates/insar_burst/readme.md.txt.j2:131]: @forrestfwilliams I think you're best suited to answer this.

File: src/hyp3_isce2/metadata/templates/insar_burst/readme.md.txt.j2 [hunk 5]
```diff
@@ -0,0 +1,306 @@
...
+
+The lv_theta map is required for applying atmospheric corrections, and for calculating vertical displacement values relative to a new phase unwrapping reference point.
+
+----------------
+
+## 7. DEM used to process the data
+
+This is a copy of the Digital Elevation Model (DEM) layer used for processing. This layer is tagged with _dem.tif, and is a 16-bit signed integer GeoTIFF.
+
+The source DEM has a geoid correction applied before it is used for processing, so height values in this file will differ from the source DEM. The DEM is down-sampled to twice the pixel spacing of the desired InSAR output pixel spacing to smooth it before processing. As such, an 80-m InSAR product requires the DEM to be down-sampled to a pixel spacing of 160 m. It is provided in the same pixel spacing/alignment and projected to the same UTM Zone as the interferogram files.
```
Comment (hjkristenson) [inline src/hyp3_isce2/metadata/templates/insar_burst/readme.md.txt.j2:209]: @forrestfwilliams Verify that this is the correct treatment of the DEM. Do we still downsample?

File: src/hyp3_isce2/metadata/templates/insar_burst/readme.md.txt.j2 [hunk 6]
```diff
@@ -0,0 +1,305 @@
...
+
+The GeoTIFF contains the unwrapped phase difference values for each pixel, corresponding to the change in distance along the line of sight of the sensor. Positive values indicate movement away from the sensor (caused by subsidence or lateral movement away from the sensor) and negative values indicate movement towards the sensor (uplift or lateral movement towards the sensor). Note that deformation often includes both vertical and horizontal components, and additional processing or external reference data is required to calculate the relative contributions of vertical and lateral movement.
+
+The PNG file displays the unwrapped phase with a cyclic color ramp applied. Each color cycle (fringe) represents a phase difference of 6 pi. Because each 2-pi difference is equivalent to half the wavelength of the sensor, each 6-pi fringe represents about 8.3 cm of line-of-sight displacement for these Sentinel-1 products.
```
Comment (jtherrmann) [inline src/hyp3_isce2/metadata/templates/insar_burst/readme.md.txt.j2:159]: In this case the PNG is just grayscale.

File: src/hyp3_isce2/metadata/templates/insar_burst/readme.md.txt.j2 [hunk 7]
```diff
@@ -0,0 +1,299 @@
...
+
+The directory name for this product is: {{ reference_granule_burst_name }}x{{ secondary_granule_burst_name }}
+
+The reference_burst_name and secondary_burst_name use the following naming conventions:
+
+S1_bbbbbb_xxn_yyyymmddThhmmss_pp_cccc-BURST
+
+bbbbbb: Relative burst ID values assigned by ESA. Each value identifies a consistent burst footprint; relative burst ID values differ from one subswath to the next.
+
+xxn: Three character combination that represents the acquisition mode and subswath number. The first two characters xx represent the acquisition mode (IW or EW) and the last character identifies the subswath number (1-3 for IW, 1-5 for EW). IW mode indicates Interferometric Wideswath, which acquires a 250 km swath composed of three subswaths. EW mode indicates Extra-Wide Swath, which acquires a 400 km swath composed of 5 subswaths.
```
Comment (hjkristenson) [inline src/hyp3_isce2/metadata/templates/insar_burst/readme.md.txt.j2:31]: I edited the description and left out the resolution info for the acquisition modes. I almost left out the IW/EW descriptions (last two sentences), but left them for now. How detailed do we want to get here? Does the ability to process EW granules impact the way that the pixel spacing is calculated? All of our original documentation was geared for IW; thought I'd check to make sure there isn't unexpected fallout.

File: src/hyp3_isce2/metadata/templates/insar_burst/readme.md.txt.j2 (hunk 2 above)
Comment (hjkristenson) [reply src/hyp3_isce2/metadata/templates/insar_burst/readme.md.txt.j2:129]: We're currently just including grayscale PNGs for unwrapped interferograms.

File: src/hyp3_isce2/metadata/templates/insar_burst/readme.md.txt.j2 (hunk 7 above)
Comment (forrestfwilliams) [reply src/hyp3_isce2/metadata/templates/insar_burst/readme.md.txt.j2:31]: I'd like to keep it in.

File: src/hyp3_isce2/metadata/templates/insar_burst/readme.md.txt.j2 (hunk 6 above)
Comment (forrestfwilliams) [reply src/hyp3_isce2/metadata/templates/insar_burst/readme.md.txt.j2]: Updated the text to reflect this.

File: src/hyp3_isce2/metadata/templates/insar_burst/readme.md.txt.j2 (hunk 5 above)
Comment (forrestfwilliams) [reply src/hyp3_isce2/metadata/templates/insar_burst/readme.md.txt.j2]: I don't think this downsample happens. Removing.

**Review by forrestfwilliams — APPROVED** [review]

**Review by cirrusasf — COMMENTED** [review]
I went through the code as well as the j2 file. They look good to me.

**Pull request merged**